app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -48,7 +60,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         if (id == R.id.action_settings) {
             return true;
         }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/8d3333d0ff432d0557742aed5ff0615effcedf8c/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L62"></span>’}’ at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).</td>
    </tr>
  </tbody>
</table>